hummingbird/ml/convert.py

```diff
@@ -73,15 +73,6 @@ def _supported_backend_check_config(model, backend, extra_config):
     if backend is torch.jit.__name__ and constants.TEST_INPUT not in extra_config:
         raise RuntimeError("Backend {} requires test inputs. Please pass some test input to the convert.".format(backend))
 
-    if backend is onnx.__name__:
-        if constants.ONNX_INITIAL_TYPES not in extra_config and constants.TEST_INPUT not in extra_config:
```

Comment:
We could remove the support for `ONNX_INITIAL_TYPES`, but the onnx convert depends on it. Should we send a PR there and fix it as we did for the other fix in the API? (I am all for it since we simply our API because we remove one unnecessary extra param).

Reply:
that would be excellent if we could get that in. I can't see any reason for keeping this other than if the user needs to manually add it for some weird reason...?

Reply:
Yea, I don't know. Ok let me remove it and see what breaks.